src/skin/js/firstRun.js

```diff
@@ -47,27 +55,28 @@ $(function () {
   async function checkIsPinned() {
     let userSettings = await chrome.action.getUserSettings();
     if (userSettings.isOnToolbar) {
-      hideNudgeOverlay();
+      hideNudge();
       clearInterval(interval_id);
     }
   }
 
   function renderNudge() {
+    // switch the instructional graphic for Opera
+    if (window.navigator.userAgent.match(/OPR\//)) {
+      $('#pin-image').attr("src", "images/pinning-instructions-opera.png");
```

Comment:
The arrow should be farther to the right on Opera:

```suggestion
      $('#pin-image').attr("src", "images/pinning-instructions-opera.png");
      $('#pin-nudge-tail').css({
          right: '10px'
      });
```
<img width="1282" alt="Screenshot 2025-03-13 at 12 33 01 PM" src="https://github.com/user-attachments/assets/00e79a04-175d-4629-b143-e46964e1562e" />